docs/APIRef.Configuration.md

```diff
@@ -63,7 +63,7 @@ Please find the [Detox example app](/examples/demo-react-native/detox.config.js)
         "build": "...",
         "type": "android.attached",
         "device": {
-          "adbName": "YOGAA1BBB412",
+          "adbName": "localhost:555[56]",
```

Comment:
I think we need to keep the previous example as-is (perhaps expand to the 'array' form) and _add_ the new option as `android.genymoition.release` right after it

Reply:
Agree. Here we can expand that to array, like `YOGAA1BBB412|YOGAA1BBB413` and add a Genymotion example separately.

Reply:
So, shall we? @d4vidi